bin/pocs_shell

```diff
@@ -75,14 +75,14 @@ class PocsShell(Cmd):
         return self.pocs.is_safe()
 
     def do_drift_align(self, *arg):
-        """Enter the drift alignment shell """
+        """Enter the drift alignment shell."""
         self.do_reset_pocs()
         print_info('*' * 80)
         i = DriftShell()
         i.cmdloop()
 
     def do_start_messaging(self, *arg):
-        """ Starts the messaging system for the POCS ecosystem
+        """Starts the messaging system for the POCS ecosystem
```

Comment:
Period here too.

I wish there were a way in reviews to make very simple edits like this...

Reply:
Done.